src/sensor/ping.h

```diff
@@ -185,23 +185,28 @@ class Ping : public PingSensor
      */
     void set_mode_auto(bool mode_auto)
     {
-        ping1D_set_mode_auto m;
+        ping1d_set_mode_auto m;
         m.set_mode_auto(mode_auto);
         m.updateChecksum();
         writeMessage(m);
-        request(Ping1DNamespace::ModeAuto);
+        request(Ping1dId::MODE_AUTO);
     }
     Q_PROPERTY(bool mode_auto READ mode_auto WRITE set_mode_auto NOTIFY modeAutoUpdate)
 
+    // sensor only supports continuous profile output
+    enum ContinuousId : uint16_t {
+        PROFILE = Ping1dId::PROFILE
```

Comment:
This doesn't look as a good approach, if `do_continuous_start` only supports one argument, it should not accept any argument at all.

Reply:
The idea is to use it to request different messages, so it is the correct design for `do_continuous_start` to have a message id parameter. The sensor just doesn't support continuous output for `distance` messages yet. When it does, we will add it to the list here.